roles/wildfly_install/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-wildfly_version: '26.0.0.Final'
+wildfly_version: '26.1.0.Final'
```

Comment:
@RanabirChakraborty for clarity purpose, I would prefer to see one commit with only the 26.0 to 26.1 bumps and another commit with the new scenario. (and another commit with the changes to the molecule/defaults).